### PR TITLE
[v9] Update desktop access docs for 9.0 (#10406)

### PR DESCRIPTION
**Pull request by zmb3**
Update desktop access docs for 9.0

- Remove admonition about being in preview
- Add audit events listing
- Break reference.mdx up into separate pages

Backports #10406

### CODE REVIEW

File: docs/pages/desktop-access/rbac.mdx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    # Desktop session recording is enabled if one or more of the user's
+    # roles has enabled recording. Defaults to true if unspecified.
+    # Desktop sessions will never be recorded if auth_service.session_recording
+    # is set to 'off' in teleport.yaml or if the cluster's session_recording_config
+    # resource has set 'mode: off'.
+    record_sessions:
+      desktop: true
+
+    # Specify whether clipboard sharing should be allowed with the
+    # remote desktop (requires a supported browser). Defaults to true
```
Comment (r0mant) [inline docs/pages/desktop-access/rbac.mdx:31]: > requires a supported browser

Are the supported browsers mentioned anywhere?

File: docs/pages/desktop-access/rbac.mdx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+`tctl users add alice --windows-logins=Administrator,DBUser`.
+
+New clusters automatically populate the preset `access` role with the following:
+
+```yaml
+allow:
+  windows_desktop_logins: ["{{internal.windows_logins}}"]
+```
+
+## Clipboard Access
```
Comment (r0mant) [inline docs/pages/desktop-access/rbac.mdx:105]: Maybe in this section would be a good place to add an admonition specifying which browsers support clipboard sharing.

File: docs/pages/desktop-access/rbac.mdx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+By default, desktop session recording is considered enabled in Teleport roles
+unless it is explicitly disabled:
+
+```yaml
+record_sessions:
+  desktop: false
+```
+
+In order to disable desktop session recording for a user, *all* of the user's
+roles must disable it. In other words, the presence of a single role which
```
Comment (r0mant) [inline docs/pages/desktop-access/rbac.mdx:137]: Nit: I think the "In other words..." sentence is superflous.

File: docs/pages/desktop-access/reference/configuration.mdx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+(!docs/pages/includes/desktop-access/desktop-config.yaml!)
+```
+
+## Deployment
+
+The Windows Desktop Service can be deployed in two modes.
+
+### Direct mode
+
+In *direct* mode, Windows Desktop Services registers directly with the Teleport
```
Comment (r0mant) [inline docs/pages/desktop-access/reference/configuration.mdx:25]:
```suggestion
In *direct* mode, Windows Desktop Service registers directly with the Teleport
```

**Review by r0mant — APPROVED** [review]

**Comment by zmb3** [comment]
Thanks for the review, @r0mant.

Since this is a backport, I'll open up a separate PR to master with your suggested improvements.

**Review by ptgott — APPROVED** [review]

**Pull request merged**